app/upgrades/v23/upgrades.go

```diff
@@ -94,7 +94,7 @@ func migrateMainnetPools(ctx sdk.Context, concentratedKeeper concentratedliquidi
 
 	// Migrate concentrated pools
 	for _, poolId := range poolIDsToMigrate {
-		if err := concentratedKeeper.MigrateAccumulatorToScalingFactor(ctx, poolId); err != nil {
+		if err := concentratedKeeper.MigrateIncentivesAccumulatorToScalingFactor(ctx, poolId); err != nil {
```

Comment:
Rename MigrateAccumulatorToScalingFactor to include the word "incentive" since there are two migrations now (incentive migrations and spread factor migrations)